app/controllers/api/v1/hook_relay_controller.rb

```diff
@@ -1,11 +1,10 @@
 class Api::V1::HookRelayController < Api::BaseController
-  before_action :set_hook_relay_report_params
```

Comment:
I felt compelled to clean this controller up a bit. Sorry for the extra review overhead.